infrastructure/ami/scripts/install-huggingface-libraries.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Activate the neuron virtual environment
+source /opt/aws_neuron_venv_pytorch/bin/activate
+
+echo "Step: install-hugging-face-libraries"
+
+echo "TRANSFORMERS_VERSION: $TRANSFORMERS_VERSION"
+echo "OPTIMUM_VERSION: $OPTIMUM_VERSION"
+
+pip install --upgrade --no-cache-dir \
+    "transformers[sklearn,sentencepiece,vision]==$TRANSFORMERS_VERSION" \
```

Comment:
Why not letting optimum-neuron pull the right version ?

Reply:
Would this install the extras? i don't think thats possible.

Reply:
I don't think it will require extras, https://github.com/huggingface/optimum-neuron/blob/main/setup.py#L16

Reply:
We require the transformers extras with `sklearn,sentencepiece,vision` to make sure training for all model works correctly, e.g. models needing sentence piece or pillow.